python/bin/xpy_dev_open

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+import argparse
+import sys
+
+import xnvme.ctypes_mapping as capi
```

Comment:
Not a hard requirement, but maybe consider `import xnvme.ctypes_mapping as xnvme`. Doesn't make much difference here, but in a larger program, it's maybe a bit ambiguous to call it `capi`.

Reply:
Sure. Changed to ``xnvme``.